Resources/Locale/en-US/seeds/seeds.ftl

```diff
@@ -75,3 +75,5 @@ seeds-watermelon-name = watermelon
 seeds-watermelon-display-name = watermelon plant
 seeds-grape-name = grape
 seeds-grape-display-name = grape plant
+seeds-coffea-name = cocoa
```

Comment:
Uh, shouldn't this be cocoa instead of coffea?

Reply:
Yes, it's just that at first I planned to make a coffee tree, but then I changed it to a cocoa tree